_posts/2018-07-04-aerogear-mobile-services-announcement.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: news
+section: news
+title: Announcing AeroGear Mobile Services on OpenShift
+author: aerogear
+---
+
+Over the past six months, the AeroGear team has been busy working on a new vision for Cloud Native Mobile and Modern App Development. Today, this vision is been realised, in the form of a set of Client SDKs and Mobile Services on OpenShift. Until now, the AeroGear project has primarily focused on Push Notifications; we're expanding the suite of services we offer today to include a number of other Mobile Services for use in your applications on OpenShift.
```

Comment:
Today, this vision is available to developers in the form...

Until now, the AeroGear project has primarily focused on Push Notifications, but now we offer a suite of services, including Push Notifications, Identity Management, to make developers more productive.